.github/workflows/ci.yml

```diff
@@ -93,6 +99,8 @@ jobs:
         uses: docker/setup-buildx-action@988b5a0280414f521da01fcc63a27aeeb4b104db # v3.6.1
       - name: download deps
         run: go mod download
+      - name: Setup QEMU
+        run: docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
would this run in arm64 agents? do we want to use tonis' image?

Reply:
This is specifically for dalec CI which only runs on amd64, and only to allow a specific test to function.